sources/web/datalab/static/datalab.js

```diff
@@ -102,7 +105,9 @@ define([
   'static/notebook-app',
   'static/notebook-list',
   'static/websocket'
-], function(ipy, events, dialog, utils, security, appbar, editapp,
+], function(ipy,     events, dialog, utils, security, appbar, editapp,
```

Comment:
That was to make the names line up between this line and the call to initializeDatalab. Since I changed IPython to ipy, I removed the spaces here.